packages/react/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -12,33 +12,55 @@ const radioStyles: ComponentSlotStylesInput<
 > = {
   root: ({ props }): ICSSInJSStyle => ({
     outline: 0,
+    display: 'inline-flex',
+    alignItems: 'center',
+    cursor: props.disabled ? 'default' : 'pointer',
     ...(!props.vertical && {
       display: 'inline-block',
     }),
   }),
 
   label: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    cursor: 'pointer',
-    display: 'inline-flex',
-    alignItems: 'baseline',
-    fontWeight: 400,
-    minHeight: '2.5rem',
     backgroundColor: 'transparent',
+    color: p.checked ? v.labelColorChecked : v.labelColorDefault,
+    ':hover': {
+      color: v.labelColorChecked,
+    },
+    ':focus': {
+      color: v.labelColorChecked,
+    },
     ...(p.disabled && {
       color: v.colorDisabled,
+      ':hover': {
+        color: v.colorDisabled,
+      },
     }),
   }),
 
   icon: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    margin: `0 ${pxToRem(10)} 0 0`,
-    color: p.checked ? v.colorChecked : v.color,
-    backgroundColor: p.checked ? v.colorBackgroundChecked : v.colorBackground,
-    borderColor: p.checked ? v.colorBorderChecked : v.colorBorder,
-    outlineColor: v.color,
-
+    margin: `0 ${pxToRem(12)} 0 0`,
+    backgroundColor: p.checked ? v.iconColorBackgroundChecked : 'transparent',
+    height: `${pxToRem(12)}`,
+    width: `${pxToRem(12)}`,
+    boxShadow: 'none',
+    borderStyle: 'solid',
+    borderWidth: `${pxToRem(1)}`,
+    borderColor: p.checked ? v.iconColorBorderChecked : v.iconColorBorderDefault,
+    ':hover': {
+      borderColor: v.iconColorBorderChecked,
```

Comment:
if we expect hover and focus behavior to apply the same style it'd be better to extract the styles in one object and reuse it